res/css/views/elements/_AddressSelector.scss

```diff
@@ -23,6 +23,7 @@ limitations under the License.
     border-radius: 3px;
     border: solid 1px $accent-color;
     cursor: pointer;
+    z-index: 1;
```

Comment:
Hmm, I don't have all the context, but I guess this just hides the bleeding? Could we rather prevent it from bleeding into the CSS of this component at all by scoping it differently?

Reply:
I had another look, fix makes sense now!